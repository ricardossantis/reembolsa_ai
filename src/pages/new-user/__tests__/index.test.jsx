import React from "react";
import renderer from "react-test-renderer";
import { useSelector } from "react-redux";
import NewUser from "../index";
//import { shallow } from "enzyme";

jest.mock("react-redux", () => ({
  useSelector: jest.fn(() => ({
    user: "edson",
    id: 1,
    auth: "token",
  })),
}));

describe("behavior", () => {
  it("retrieving redux state data", () => {
    useSelector.mockImplementation({
      user: "edson",
      id: 1,
      auth: "token",
    });
    expect(useSelector).toMatchSnapshot();
  });
});
