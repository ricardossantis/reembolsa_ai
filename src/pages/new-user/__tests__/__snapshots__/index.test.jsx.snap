// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`behavior retrieving redux state data 1`] = `[MockFunction]`;
